src/web_app/js/call.js

```diff
@@ -474,9 +503,14 @@ Call.prototype.startSignaling_ = function() {
     } else {
       this.pcClient_.startAsCallee(this.params_.messages);
     }
+    this.maybeReportGetUserMediaErrors_();
   }.bind(this))
   .catch(function(e) {
-    this.onError_('Create PeerConnection exception: ' + e.message);
+    this.onError_('Create PeerConnection exception: ' + e);
+    // Let the callstats backend know we failed connecting to the
+    // other endpoint.
+    this.pcClient_.sendCallstatsEvents('fabricFailed');
```

Comment:
actually, this one is complicated since it's not clear that PC client even exists at this point.